Triangle.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    class Triangle: Point
+    {
+
+        private double x1;
+        private double y1;
+        private double x2;
+        private double y2;
+
+
+        public Triangle(double x0, double y0, double x1, double y1, double x2, double y2):base(x0,y0)

```

Comment:
this class doesn't have an implementation to change coordinates if you need it. Read about get; set; and why do people use it

Reply:
Check all your classes and think about what properties should be public and what should be private